### DISCUSSION TITLE
Shimmer Firefly password.... Need your help fellows

### DISCUSSION THREAD

**Discussion by JDZ06** [iotaledger/firefly#8023] [Help & Support]
Hi, i usually log in to my shimmer firefly account to update the last version, and turns out my password is wrong. I thought it was the same than the Firefly Iota one, but nope....
There is no "forgot your PS" link and i searched on the different topics here but haven't found something relevant...  I've got seed phrase and everything.  Please tell me like if i was dumb, excactly what to do and where to go pls.

Really appreciate your answers in advance

**Comment by begonaalvarezd** [comment]
If you dont remember your pin (6 digit pin to enter the profile) you can create a new profile restoring from your 24 words mnemonic or your latest stronghold
If you dont remember your stronghold password you can restore using your 24 words mnemonic

**Reply by JDZ06** [reply]
Hi :)
I do have my 6 digit pin code. Tell me precisely where i need to go because the shimmer firefly page looks blurry and empty.
Thanks buddy in advance for your help

**Reply by begonaalvarezd** [reply]
In the welcome screen just press "+" to create a new profile and follow the steps importing from stronghold(and password to decrypt)/ledger/24 words, you need at least 1 of those in order to be able to restore your profile

**Reply by JDZ06** [reply]
Thanks, that helped me a lot.... Now how should i get my SMR from my first locked up acount ?
